index.html

```diff
@@ -1,11 +1,19 @@
 <!DOCTYPE html>
 <html lang="nl">
+<!-- door Jiro -->
 <head>
     <meta charset="UTF-8">
     <link rel="stylesheet" href="styles.css">
     <title>Les 3 - Opdracht CSS Grid</title>
 </head>
 <body>
-
+    <div class="container">
+        <aside class="grid-item a aside-small">a</aside>
```

Comment:
kleine letters bij class en id = goed ! (zelf had ik een hoofdletter gekozen bij ID maar dat is dus fout haha)